pymongoexplain/explainable_collection.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020-present MongoDB, Inc.
```

Comment:
We also need to add a `pymongoexplain/__init__.py` and `test/__init__.py` (they can be empty for now). For the reason why check out https://docs.python.org/3/tutorial/modules.html#packages

Reply:
Done.